spring-boot-security/src/test/java/com/baeldung/springbootsecurity/oauth2server/OAuth2IntegrationTestSupport.java

```diff
@@ -31,4 +39,18 @@ protected OAuth2RestTemplate getOAuth2RestTemplate(final ClientCredentialsResour
         restTemplate.setMessageConverters(singletonList(new MappingJackson2HttpMessageConverter()));
         return restTemplate;
     }
+
+    protected String executeGetRequest(OAuth2RestTemplate restTemplate, String path) {
+        return restTemplate.execute(base.toString() + path, GET, getRequestCallback(), getStringResponseExtractor());
+    }
+
+    protected RequestCallback getRequestCallback() {
+        return clientHttpRequest -> {
+        };
+    }
```

Comment:
This is a do nothing callback... I would suggest that this way of expressing it is a bit harder to parse. If you did something like this:

```java
private static final RequestCallback DO_NOTHING_CALLBACK = request -> {};
```

And then used it at the call site as `DO_NOTHING_CALLBACK` - I think it might be more self-documenting.